configuration/src/main/kotlin/com/malinskiy/marathon/config/strategy/ExecutionStrategyConfiguration.kt

```diff
@@ -0,0 +1,31 @@
+package com.malinskiy.marathon.config.strategy
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * @property fast fail-fast or success-fast depending on the value of [mode]. This doesn't affect the result
+ *  of the run, it only saves compute time
+ */
+data class ExecutionStrategyConfiguration(
+    @JsonProperty("mode") val mode: ExecutionMode = ExecutionMode.ANY_SUCCESS,
+    @JsonProperty("fast") val fast: Boolean = true,
```

Comment:
This is explained in the docs, short version is you might want to run other retries anyways in some scenarios because test status is not the only output of a test, test probability of passing is one of the examples

Reply:
Just read it. Totally makes sense to me. 👍